gobblin-core/src/test/java/gobblin/converter/csv/PercentageDoubleStringConverterTest.java

```diff
@@ -0,0 +1,36 @@
+package gobblin.converter.csv;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import gobblin.converter.DataConversionException;
+
+
+@Test(groups = {"gobblin.converter"})
+public class PercentageDoubleStringConverterTest {
+  public void convertOutput()
+      throws IOException, DataConversionException {
+    JsonParser parser = new JsonParser();
+    JsonElement jsonElement = parser
```

Comment:
Oh I see. I didn't add that file. This is the file I added. 10_fields_with_percentage.json